conf/membercontributions.routes

```diff
@@ -15,10 +15,13 @@ POST        /:srn/total-member-contribution/:index                      controll
 GET         /:srn/change-total-member-contribution/:index               controllers.nonsipp.membercontributions.TotalMemberContributionController.onPageLoad(srn: Srn, index: Max300, mode: Mode = CheckMode)
 POST        /:srn/change-total-member-contribution/:index               controllers.nonsipp.membercontributions.TotalMemberContributionController.onSubmit(srn: Srn, index: Max300, mode: Mode = CheckMode)
 
-GET         /:srn/check-answers-member-contributions/:index             controllers.nonsipp.membercontributions.MemberContributionsCYAController.onPageLoad(srn: Srn, index: Max300, mode: Mode = NormalMode)
-POST        /:srn/submit-check-answers-member-contributions/:index      controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = NormalMode)
-GET         /:srn/change-member-contributions/:index                    controllers.nonsipp.membercontributions.MemberContributionsCYAController.onPageLoad(srn: Srn, index: Max300, mode: Mode = CheckMode)
-POST        /:srn/submit-change-member-contributions/:index             controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = CheckMode)
+GET         /:srn/check-answers-member-contributions/:index                                     controllers.nonsipp.membercontributions.MemberContributionsCYAController.onPageLoad(srn: Srn, index: Max300, mode: Mode = NormalMode)
+POST        /:srn/submit-check-answers-member-contributions/:index                              controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = NormalMode)
+GET         /:srn/change-member-contributions/:index                                            controllers.nonsipp.membercontributions.MemberContributionsCYAController.onPageLoad(srn: Srn, index: Max300, mode: Mode = CheckMode)
+POST        /:srn/submit-change-member-contributions/:index                                     controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = CheckMode)
+POST        /:srn/change-submit-check-answers-member-contributions/:index                                controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = ViewOnlyMode)
+GET         /:srn/check-answers-member-contributions/:index/view/:year/:current/:previous                controllers.nonsipp.membercontributions.MemberContributionsCYAController.onPageLoadViewOnly(srn: Srn, index: Max300, mode: Mode = ViewOnlyMode, year: String, current: Int, previous: Int)
+GET         /:srn/submit-check-answers-member-contributions/view/:year/:current/:previous         controllers.nonsipp.membercontributions.MemberContributionsCYAController.onSubmitViewOnly(srn: Srn, year: String, current: Int, previous: Int)
```

Comment:
`:previous` may become unused here, dependently on the code changes you do to address other comments